internal/backup_push_handler.go

```diff
@@ -69,59 +69,18 @@ func getDeltaConfig() (maxDeltas int, fromFull bool) {
 	return
 }
 
-// TODO : unit tests
-// HandleBackupPush is invoked to perform a wal-g backup-push
-func HandleBackupPush(uploader *Uploader, archiveDirectory string, isPermanent bool, isFullBackup bool) {
-	archiveDirectory = utility.ResolveSymlink(archiveDirectory)
-	maxDeltas, fromFull := getDeltaConfig()
-	checkPgVersionAndPgControl(archiveDirectory)
-	var err error
-	var previousBackupSentinelDto BackupSentinelDto
-	var previousBackupName string
-	incrementCount := 1
-
+func createAndPushBackup(
+	uploader *Uploader,
+	archiveDirectory, backupsFolder,
+	previousBackupName string, previousBackupSentinelDto BackupSentinelDto,
```

Comment:
Move "previousBackupName" to line with "archiveDirectory, backupsFolder," it will be easier to read